src/data-structures/games.js

```diff
@@ -8,8 +8,8 @@ const games = [
       released: '10th of December, 2020',
       platforms: 'Playstation 4, PC, Xbox Series S/X, Playstation 5, Xbox One',
       mainGenre: 'RPG',
-      Developers: 'CD PROJECT RED, CD PROJECT',
-      Publishers: 'CD PROJECT RED',
+      developers: 'CD PROJECT RED, CD PROJECT',
```

Comment:
I think you should remove the `more` key as it only makes sense for the current case. If that data will be displayed somewhere elese in some other form it doesn't make sense to be under `more`